operations/app/src/modules/prime_data_hub/main.tf

```diff
@@ -0,0 +1,19 @@
+locals {
+    location = "eastus"
+}
+
+module "storage" {
```

Comment:
This is fantastic! I love the way you're breaking up each part of the environment into a separate module. This will help keep our Terraform configuration sane.